libs/init.js

```diff
@@ -54,32 +49,40 @@ module.exports = (app) => {
 
     app.logger.info('Init -> Configure datadog logger');
     const middlewareLogger = function (req, res, next) {
+
         exp.logger = (id, title, info) => {
-            app.logger.warn(
-                'HASH: ' + req.method + ' ' + req.originalUrl + ': ' + title,
-                {
-                    type: 'malicious',
-                    templateId: id,
-                    info,
-                    http: {
-                        client_ip: req.ip,
-                        host: req.headers.host,
-                        method: req.method,
-                        path: req.path,
-                    },
-                    request: {
-                        query: req.query || {},
-                        params: req.params || {},
-                        body: req.body || {},
-                        headers: {
-                            ...req.headers,
-                            ...{
-                                cookie_parsed: req.cookies,
-                            },
+            const payload = {
```

Comment:
Is there a field we can add to further indicate this is from a honeypot, or will the service provide a good label for filtering?

Reply:
The requirement I got from Andy's before that they are using service name / env to filter honeypot data.